src/fonduer/utils/fonduer_model.py

```diff
@@ -0,0 +1,398 @@
+import logging
```

Comment:
This is good start.

One high-level comment here is: can restructure this giant file into a module of fonduer (e.g. packaging) and split this file into small files?

Reply:
I agree with moving this into `fonduer.packaging`.
I think I can move `F_matrix` and `L_matrix` into other files, but this would be the best thing I could do.
It seems to be the convention to have all the functions (e.g., `log_model`, `save_model`, wrapper, `_get_default_conda_env`) in a single file.
See https://github.com/mlflow/mlflow/blob/7312255f38a8c06181a9e3dfc0e9e08b45e21ea6/mlflow/pytorch/__init__.py

Let me see what I can do.

Reply:
I merged `F_matrix` (now `FonduerModel.convert_features_to_matrix`) with `fonduer.utils.utils_udf.get_sparse_matrix` to avoid duplicated codes.
fonduer_model.py still has 400+ lines of codes, but each function is not so long (the longest `save_model` has 70L without signature and docstring).